spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -66,3 +66,6 @@ definitions:
                   values:
                       - 0: System Healthy
                       - 1: An error has occurred
+        - protocol:
```

Comment:
We've currently got a bit of a mix of encoding vs raw strings for versioning info throughout Piksi. It would probably be good to standardize this. Off the top of my head:

* STM FW Version
  * String
* NAP FW Version
  * String
  * Note: Older versions were a 4 byte identifier
* Bootloader Version
  * String
  * Note: Older versions were a 1 byte identifier
* HW Version
  * Currently a 4 byte identifier. Should we change this to a string?
* SBP Protocol Version
  * This PR would introduce it as a string